test/README.md

```diff
@@ -0,0 +1,57 @@
+# Testing 
+
+## Overview
+
+We maintain a collection of `.cff` files for testing. In these files we try to cover
+various possible scenarious described in the specification. During the test, these
+files are validated using `cffconvert` (a command line utility to read a `.cff` file
+and convert it to BibTex and other formats). We check that all files are validated or
+rejected as expected.
+
+## Dependencies
+
+- Python 3.6 or higher
+- [cffconvert](https://pypi.org/project/cffconvert/) version 1.0.0 or higher
+- [pytest](https://pypi.org/project/pytest/)
+
+You can install `cffconvert` and `pytest` using the `requirements.txt` file from
+this directory by entering the following command:
+
+    python3 -m pip install -r requirements.txt
```

Comment:
```suggestion
    python3 -m pip install -r test/requirements.txt
```
(just to clarify which `requirements.txt` we mean, the other one from `/` may disappear later)